src/main/java/eu/bbmri_eric/negotiator/database/model/Network.java

```diff
@@ -0,0 +1,99 @@
+package eu.bbmri_eric.negotiator.database.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import lombok.*;
+
+@Getter
+@Setter
+@Entity
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class Network {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "resource_id_seq")
+  @SequenceGenerator(name = "resource_id_seq", initialValue = 10000, allocationSize = 1)
+  private Long id;
+
+  @NotNull private String uri;
+
+  /**
+   * The name of the network.
+   *
+   * @param name the name of the network
+   * @return the name of the network
+   */
+  @Column(unique = true)
+  private String name;
+
+  /**
+   * A unique and persistent identifier issued by an appropriate institution.
+   *
+   * @param externalId the external identifier of the network
+   * @return the external identifier of the network
+   */
+  @NotNull
+  @Column(unique = true)
+  private String externalId;
+
+  /**
+   * The contact email of the network.
+   *
+   * @param contactEmail the contact email of the network
+   * @return the contact email of the network
+   */
+  private String contactEmail;
+
+  /**
+   * The managers of the network.
+   *
+   * @param managers the managers of the network
+   * @return the managers of the network
+   */
+  @ManyToMany(mappedBy = "networks")
+  private Set<Person> managers = new HashSet<>();
```

Comment:
currently the class exposes public a setter for the managers as well as add/remove methods on Person. Is this desired behviour? I think this can lead to some bugs.